2025-04-07 12:40:41.784 UTC [16713] LOG:  starting PostgreSQL 16.1 on x86_64-conda-linux-gnu, compiled by x86_64-conda-linux-gnu-cc (conda-forge gcc 12.3.0-3) 12.3.0, 64-bit
2025-04-07 12:40:41.784 UTC [16713] LOG:  listening on IPv6 address "::1", port 5432
2025-04-07 12:40:41.784 UTC [16713] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-07 12:40:41.785 UTC [16713] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2025-04-07 12:40:41.788 UTC [16716] LOG:  database system was shut down at 2025-04-07 12:40:33 UTC
2025-04-07 12:40:41.794 UTC [16713] LOG:  database system is ready to accept connections
2025-04-07 12:42:31.443 UTC [17677] FATAL:  role "user-theo15" does not exist
2025-04-07 12:42:53.324 UTC [17840] FATAL:  role "postgres" does not exist
2025-04-07 12:45:41.874 UTC [16714] LOG:  checkpoint starting: time
2025-04-07 12:45:46.001 UTC [16714] LOG:  checkpoint complete: wrote 44 buffers (0.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=4.124 s, sync=0.002 s, total=4.128 s; sync files=11, longest=0.001 s, average=0.001 s; distance=256 kB, estimate=256 kB; lsn=0/14C0A90, redo lsn=0/14C0A58
